src/components/navigation/Breadcrumbs/Breadcrumb.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import BreadcrumbAction from './BreadcrumbAction';
+import Icon from '../../base/icons/Icon';
+
+function Breadcrumb({ breadcrumbClasses, ...props }) {
+  const children = props.children;
+
+  return (
+        <div className={breadcrumbClasses}>
+            {React.Children.map(children, (child, i) => (
+                <span>
+                    {i > 0 ? <Icon name="chevron-right" /> : ''}

```

Comment:
`{i > 0 && <Icon name="chevron-right" />}` is an inline shortcut without needing the empty string